Added: alt attribute to few images (#19669)
mdn/content: files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md

As we said above, all drawing operations are done by manipulating a {{domxref("CanvasRenderingContext2D")}} object (in our case, `ctx`). Many operations need to be given coordinates to pinpoint exactly where to draw something — the top left of the canvas is point (0, 0), the horizontal (x) axis runs from left to right, and the vertical (y) axis runs from top to bottom.

> ![A graph with small squares covering it's area which is considered as a canvas that has x axis and y axis. The top left corner of the canvas is point (0, 0). The horizontal (x) axis runs from left to right, and the vertical (y) axis runs from top to bottom. The center of the canvas has a quadrilateral filled with steelblue color. The top left corner of the quadrilateral is distanced at x units from y axis and y units from x axis. The height is along the y axis and the width is along the x axis.](canvas_default_grid.png)

Reviewer: ```suggestion
![Gridded graph paper with small squares covering its area with a steelblue square in the middle. The top left corner of the canvas is point (0, 0) of the canvas x-axis and y-axis. . The horizontal (x) axis runs from left to right denoting the width, and the vertical (y) axis runs from top to bottom denotes the height. The top left corner of the blue square is labeled as being a distance of x units from the y-axis and y units from the x-axis.](canvas_default_grid.png)
```

    - The side next to the 60 degree angle is called the **adjacent** — which we know is 50 pixels, as it is half of the line we just drew.
    - The side opposite the 60 degree angle is called the **opposite**, which is the height of the triangle we want to calculate.

>     ![A triangle pointing downwards. A dotted line splits the triangle into two equal right-angled triangles. The angle between the top and and right side of the triangle is 60 degrees. The dotted line perpendicular to the top side of the inverted triangle and opposite to the 60 degrees angle intersects the bottom vertice of the inverted triangle. This line is called as the opposite side. A section of the side of the triangle joining the point of intersection of the opposite line with the top side of the triangle to the 60 degrees angle is called adjacent side. The right side of the triangle is called as hypotenuse and it is the longest side.](trigonometry.png)

Reviewer: ```suggestion
    ![A equilateral triangle pointing downwards with labeled angles and sides. The horizontal line at the top is labeled 'adjacent'. A perpendicular dotted line, from the middle of the adjacent line, labeled 'opposite', splits the triangle creating two equal right triangles.  The right side of the triangle is labeled the hypotenuse, as it is the hypotenuse of the right triangle formed by the line labeled 'opposite'. while all three-sided of the triangle are of equal length, the hypotenuse is the longest side of the right triangle.](trigonometry.png)
```

    Let's explain the spritesheet image (which we have respectfully borrowed from Mike Thomas' [Walking cycle using CSS animation](https://codepen.io/mikethomas/pen/kQjKLW) CodePen). The image looks like this:

>     ![There are six images of a pixel character resembling a walking person from his/her right side at different instances of a single step forward. The character wears a white shirt with sky blue buttons, black trousers and black shoes. The whole sheet is called a sprite sheet. Each image is called a sprite. Each sprite is 102 pixels wide and 148 pixels high.](walk-right.png)

Reviewer: ```suggestion
    ![A sprite sheet with six sprite images of a pixelated character resembling a walking person from their right side at different instances of a single step forward. The character has a white shirt with sky blue buttons, black trousers, and black shoes. Each sprite is 102 pixels wide and 148 pixels high.](walk-right.png)
```
